lua/lspconfig/server_configurations/psalm.lua

```diff
@@ -1,9 +1,8 @@
 local util = require 'lspconfig.util'
 
-
```

Comment:
The lint step was failing in this file, so I reformatted the code. Not related to my original change.